File: SofiForce.Web/ClientApp/src/app/core/Models/ExportModels/ExportSalesHeaderModel.ts
export interface ExportSalesHeaderModel {
    salesCode: string;
    invoiceCode: string;
    invoiceDate: Date | null;
    invoiceType: string;
    clientCode: string;
    clientName: string;
    branchCode: string;
    branchName: string;
    storeCode: string;
    storeName: string;
    salesManCode: string;
    salesManName: string;
    paymentTerm: string;
    salesTime: Date | null;
    salesChannel: string;
    salesPool: string;
    itemTotal: number | null;
    offerDiscount: number | null;
    taxTotal: number | null;
    cashDiscount: number | null;
    cashPercentage: number | null;
    netTotal: number | null;
    openValue: number | null;
}

File: SofiForce.Models/Models/ExportModels/ExportSalesHeaderModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class ExportSalesHeaderModel
    {
        public string SalesCode { get; set; }
        public string InvoiceCode { get; set; }
        public DateTime? InvoiceDate { get; set; }
        public string InvoiceType { get; set; }
        public string ClientCode { get; set; }
        public string ClientName { get; set; }
        public string BranchCode { get; set; }
        public string BranchName { get; set; }
        public string StoreCode { get; set; }
        public string StoreName { get; set; }
        public string SalesManCode { get; set; }
        public string SalesManName { get; set; }
        public string PaymentTerm { get; set; }
        public DateTime? SalesTime { get; set; }
        public string SalesChannel { get; set; }
        public string SalesPool { get; set; }
        public decimal? ItemTotal { get; set; }
        public decimal? OfferDiscount { get; set; }
        public decimal? TaxTotal { get; set; }
        public decimal? CashDiscount { get; set; }
        public decimal? CashPercentage { get; set; }


        
        public decimal? NetTotal { get; set; }
        public decimal? OpenValue { get; set; }

        public string ClaimNo { get; set; }


    }
}

